src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +163,180 @@ export default async function() {
 
 Your development server will automatically restart your database whenever this file changes, regenerating your types and seeding your development data from `seed.ts`.
 
+## Connect a libSQL database for production
+
+Astro DB can connect to any local libSQL database or to any server that exposes the libSQL remote protocol, whether managed or self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL DB
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server. This is required for remote databases, not needed for local DBs like files or in-memory databases
+
+The way you get these values varies between services, but the next section will show you how you can get started with [Turso](https://turso.tech).
+
+Turso is the company behind [libSQL](https://github.com/tursodatabase/libsql), the open-source fork of SQLite that powers Astro DB. They created and maintain the libSQL project, making them the perfect choice to provide a fully managed libSQL database platform.
+
+### Getting started with Turso
+
+You can either use the CLI or the web interface to create and retrieve the values you need.
+The steps that are shown below will walk you through the process of installing the Turso CLI, logging in (or signing up), creating a new database, getting the two values we need and pushing our schema to the remote database.
```

Comment:
We should avoid `we`/`our` while guiding the user. I also slightly reworded the sentence:
```suggestion
The steps below will guide you through the process of installing the Turso CLI, logging in (or signing up), creating a new database, obtaining the two necessary values, and pushing the schema to the remote database.
```
I even wonder if we really need to specify the step concerning obtaining the values:

```suggestion
The steps below will guide you through the process of installing the Turso CLI, logging in (or signing up), creating a new database, and pushing the schema to the remote database.
```
I think it is better to inform about important steps not the intermediaries.

Reply:
I often use "we" without a specific reason, and Sarah even added an image for this lol.

![image](https://github.com/user-attachments/assets/90fd03e0-cfbe-4390-b510-f7d84c304122)

My thought on outlining the steps is that if someone wishes to use the website, they can follow the process. Although it's much simpler with the CLI, and obtaining a token via the website requires navigating through submenus, etc., I believe we should still communicate the steps to inform users of what to expect.

I do think we need more people to weigh in on this.